packages/storage-ui/src/Components/Pages/CidsPage.tsx

```diff
@@ -53,6 +54,28 @@ const CidsPage = () => {
   const classes = useStyles()
   const { pins } = useStorage()
   const [addCIDOpen, setAddCIDOpen] = useState(false)
+  const [column] = useState<"size" | "date_uploaded">("date_uploaded")
+  const [direction] = useState<"ascend" | "descend" | undefined>("descend")
+
+  const sortedPins: PinStatus[] = useMemo(() => {
+    let temp = []
+
+    switch (column) {
+    default: {
+      temp = pins.sort((a, b) => (a.created < b.created ? -1 : 1))
+      break
+    }
+    case "size": {
+      temp = pins.sort((a, b) => (a.info?.size < b.info?.size ? -1 : 1))
+      break
+    }
+    case "date_uploaded": {
+      temp = pins.sort((a, b) => (a.created < b.created ? -1 : 1))
+      break
+    }
+    }
```

Comment:
This is duplicated, we should have default at the end, and remove the "date_uploaded" part.
```suggestion
    case "size": {
      temp = pins.sort((a, b) => (a.info?.size < b.info?.size ? -1 : 1))
      break
    }
    default: {
      temp = pins.sort((a, b) => (a.created < b.created ? -1 : 1))
      break
    }
    }
```